nginx/nginx.conf.dev.template

```diff
@@ -1,5 +1,7 @@
 worker_processes 1;
 
+# see https://www.digitalocean.com/community/tutorials/nginx-access-logs-error-logs#nginx-error-log-severity-levels for configuring
+error_log  logs/error.log  info;
```

Comment:
This last commit introduces a bug when trying to lunch the nginx container
```
iscscfr-nginx-1          | nginx: [emerg] open() "/etc/nginx/logs/error.log" failed (2: No such file or directory)
```

Reply:
fixed in [7351f76](https://github.com/iScsc/iscsc.fr/pull/86/commits/7351f76b6a6753f740f427b580e6c39d9c320981)

but as stated in the commit body I'll soon open a PR to use these